interfaces/apparmor/apparmor_test.go

```diff
@@ -70,6 +73,22 @@ apparmor_parser output:
 		"--replace --write-cache -O no-expr-simplify --cache-loc=/var/cache/apparmor /path/to/snap.samba.smbd"})
```

Comment:
This is okay (in this case) because we don't divert the root directory. I can add the diversion to ensure we are indeed using the variable name.